### PR TITLE
[All Hosts] (manifest) how to control tab bar icons

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
Added an additional use of the IconUrl element and other improvements to the article.

### CODE REVIEW

File: docs/manifest/iconurl.md [hunk 1]
```diff
@@ -29,8 +29,24 @@ Specifies the URL of the image that is used to represent your Office Add-in in t
...
+For a mail add-in, the icon is displayed in the **File** > **Manage add-ins** UI (Outlook) or **Settings** > **Manage add-ins** UI (Outlook on the web). For a content or task pane add-in, the icon is displayed in the **Insert** > **Add-ins** UI. 
+
+The image is also used on the vertical tab bar for the task pane when more than one task pane is open. The tab bar appears beside the task pane whenever a second task pane is opened, regardless of whether it is a task pane in the same add-in or a different add-in. The following image shows the tab bar when the [Script Lab](/office/dev/add-ins/overview/explore-with-script-lab.md) add-in and another add-in have both been started and both the **Code** and **Run** task pane pages of Script Lab have been opened.
+
+:::image type="content" source="/javascript/api/images/built-in-vertical-tab-bar.png" alt-text="Screenshot of the top inch of a task pane, with three square tabs to the right of the upper right corner. Two of the tabs have the Script Lab icon. The third has a different icon.":::
```
Comment (samantharamon) [inline docs/manifest/iconurl.md:36]:
```suggestion
:::image type="content" source="/javascript/api/images/built-in-vertical-tab-bar.png" alt-text="The top inch of a task pane, with three square tabs to the right of the upper right corner. Two of the tabs have the Script Lab icon. The third has an icon for a different add-in.":::
```

Comment (samantharamon) [reply docs/manifest/iconurl.md:36]: Per the style guide, removing "Screenshot of" from alt text to improve SEO.

**Review by samantharamon — APPROVED** [review]
Added minor suggestion, but looks good otherwise.

File: docs/manifest/iconurl.md [hunk 2]
```diff
@@ -29,8 +29,24 @@ Specifies the URL of the image that is used to represent your Office Add-in in t
 
 ## Remarks
 
-For a mail add-in, the icon is displayed in the **File** > **Manage add-ins** UI (Outlook) or **Settings** > **Manage add-ins** UI (Outlook on the web). For a content or task pane add-in, the icon is displayed in the **Insert** > **Add-ins** UI. For all add-in types, the icon is also used in [AppSource](https://appsource.microsoft.com), if you publish your add-in to AppSource.
+For a mail add-in, the icon is displayed in the **File** > **Manage add-ins** UI (Outlook) or **Settings** > **Manage add-ins** UI (Outlook on the web). For a content or task pane add-in, the icon is displayed in the **Insert** > **Add-ins** UI. 
+
+The image is also used on the vertical tab bar for the task pane when more than one task pane is open. The tab bar appears beside the task pane whenever a second task pane is opened, regardless of whether it is a task pane in the same add-in or a different add-in. The following image shows the tab bar when the [Script Lab](/office/dev/add-ins/overview/explore-with-script-lab.md) add-in and another add-in have both been started and both the **Code** and **Run** task pane pages of Script Lab have been opened.
```
Comment (ElizabethSamuel-MSFT) [inline docs/manifest/iconurl.md:34]:
```suggestion
The image is also used on the vertical tab bar for the task pane when more than one task pane is open. The tab bar appears beside the task pane whenever a second task pane is opened, regardless of whether it is a task pane in the same add-in or a different add-in. The following image shows the tab bar when the [Script Lab](/office/dev/add-ins/overview/explore-with-script-lab) add-in and another add-in have both been started and both the **Code** and **Run** task panes of Script Lab have been opened.
```

**Pull request merged**